testing/src/cypress/custom-command/institution/common-helper-functions.cy.ts

```diff
@@ -31,16 +42,26 @@ export default class InstitutionHelperActions {
     return Cypress.env("TEST").USER_PASSWORD_SINGLE_LOCATION;
   }
 
+  getUserNameForApiTest() {
+    return Cypress.env("TEST").USERNAME_API_TEST;
+  }
+
+  getUserPasswordForApiTest() {
+    return Cypress.env("TEST").USER_PASSWORD_API_TEST;
+  }
+
   getUserDetailsSingleLocation() {
-    return Cypress.env("USER_DETAILS").USER_SINGLE_LOCATION;
+    return UserData.userDetailsSingleLocation;
```

Comment:
It is still not clear to be the strategy to get this type of information per environment.
For instance, if we have different users for differents envs, how these configs are going to work?
Is it something that we would be taking care of in upcoming PRs?

Reply:
Yeah. i ll add that approach as part of separate PR